CHANGELOG.md

```diff
@@ -3,7 +3,14 @@
 ## Fixes
 
 - [Pull request #1104: Change visually hidden footer title](https://github.com/alphagov/govuk-prototype-kit/pull/1104)
-- [Pull request #1076: Update to Node.js v14.17.6 when using Node Version Manager](https://github.com/alphagov/govuk-prototype-kit/pull/1076)
+
+### Updating Node.js
+
+We are recommending that all users of the GOV.UK Prototype Kit update to the latest version of Node.js 14 LTS, to make sure they are protected from a [recent security vulnerability in the Node package manager npm](https://github.blog/2021-09-08-github-security-update-vulnerabilities-tar-npmcli-arborist/).
+
+Please follow the [instructions on the prototype kit website](https://govuk-prototype-kit.herokuapp.com/docs/install/requirements.md#nodejs-version-14-lts) to make sure you are using Node.js version 14.17.6 or later.
+
+If you are using Node Version Manager (nvm), you can if you wish instead [run `nvm install` to install v14.17.6](https://github.com/alphagov/govuk-prototype-kit/pull/1076).
```

Comment:
```suggestion
If you are using the Node Version Manager (nvm), you can instead [run `nvm install` to install v14.17.6](https://github.com/alphagov/govuk-prototype-kit/pull/1076).
```

Reply:
Small nitpick, but I think with nvm we should either go with 'the Node version manager `nvm`,' or 'Node Version Manager' without the 'the'. Writing 'the Node Version Manager' looks wrong to me, because it's the proper name of the script.

Reply:
Ah gotcha, yeah that makes sense.

```suggestion
If you are uNode Version Manager (nvm), you can instead [run `nvm install` to install v14.17.6](https://github.com/alphagov/govuk-prototype-kit/pull/1076).
```